README.md

```diff
@@ -28,7 +28,7 @@ Hangar.do_not_delete = %w(very_important_things valuable_bitcoin_keys spatial_re
 
 ### Specifying DatabaseCleaner deletion strategy
 
-Not all ORMs support the deletion strategy, such as mongoid. You can override the clean strategy being used by the `DELETE /` request as follows:
+Not all ORMs support the deletion strategy, such as mongoid. You can override the clean strategy being used by the `DELETE /factori` request as follows:
```

Comment:
@bradhaydon is this a typo?

Reply:
You're correct that is a typo; good catch! I have pushed up an updated README.md file.